src/acp/matcher.ts

```diff
@@ -823,8 +824,9 @@ export function getClientAll(matcher: Matcher): WebId[] {
  * @param matcher The Matcher for which clients are set.
  * @param client The Client the Matcher should apply to.
  * @returns A copy of the input Matcher, applying to a different set of Clients.
- * @since Not released yet.
+ * @deprecated.
```

Comment:
Why deprecate this? I think we have similar functions to set a user on a matcher too. I don't think it's problematic.

Reply:
If the since says it's not yet released, couldn't we just wholesale delete it? Or did someone forget to mark it as released?

Reply:
It is part of the `acp_v4` export, so it is part of our public API. I wasn't sure about this (as the commit history shows ^^), but I expect it may be a common use case to want to enforce only the current client has access to a given resource, mostly for config/indexing stuff. Even if it isn't something we want to encourage overall, having a consistent way of doing so, and to interact with triples in general, with `get`, `set`, `add` and `remove` for the `<s> <p> ?o` operations.